src/Microsoft.AspNetCore.WebSockets.Protocol/project.json

```diff
@@ -11,14 +11,18 @@
     "warningsAsErrors": true,
     "keyFile": "../../tools/Key.snk",
     "nowarn": [
-      "CS1591"
+      "CS1591",
+      "CS1572",
+      "CS1573"
```

Comment:
No, but we could file them I guess. They are here because they are caused by the corefx code, so as long as that code is in our repo I don't know that we'll fix them (they're related to XML Docs, and these are internal APIs... it's kinda annoying that they are triggering at all...

I could add an issue to track removing this when we remove the corefx code and use a public API.